src/models/activityReport.js

```diff
@@ -146,6 +146,8 @@ export default (sequelize, DataTypes) => {
     },
     version: {
       type: DataTypes.INTEGER,
+      allowNull: false,
+      defaultValue: 1,
```

Comment:
Default should be 2

Reply:
OK, I will update the model and the database to reflect that